File: app/src/main/java/knf/kuma/custom/WrapWebView.java
package knf.kuma.custom;

import android.content.Context;
import android.util.AttributeSet;
import android.webkit.WebView;

public class WrapWebView extends WebView {
    public WrapWebView(Context context) {
        super(context);
    }

    public WrapWebView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public WrapWebView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public WrapWebView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }

    public WrapWebView(Context context, AttributeSet attrs, int defStyleAttr, boolean privateBrowsing) {
        super(context, attrs, defStyleAttr, privateBrowsing);
    }
}

File: app/src/main/java/knf/kuma/custom/WrapWebView.kt
package knf.kuma.custom

import android.content.Context
import android.content.res.Configuration
import android.os.Build
import android.util.AttributeSet
import android.webkit.WebView

class WrapWebView : WebView {
    constructor(context: Context) : super(context.configurated)

    constructor(context: Context, attrs: AttributeSet) : super(context.configurated, attrs)

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context.configurated, attrs, defStyleAttr)
}

val Context.configurated: Context
    get() = if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M)
        createConfigurationContext(Configuration())
    else this
